lib/metasploit/framework/data_service/stubs/credential_data_service.rb

```diff
@@ -0,0 +1,18 @@
+module LoginDataService
```

Comment:
The stub module name appears to be copy-pasta from `stubs/login_data_service.rb`